mmv1/templates/terraform/constants/iam_workforce_pool.go.erb

```diff
@@ -0,0 +1,37 @@
+const workforcePoolIdRegexp = `^[0-9a-z-]+$`
+
+func validateWorkforcePoolId(v interface{}, k string) (ws []string, errors []error) {
```

Comment:
Do we need this much effort client-side to validate names?

Typically we avoid adding too much validation client-side and let the server validate fields like this, as validation tends to change over time. If the validation changes server-side, clients don't need to change to use it, but if it's client-side and needs to change it causes delays in users being able to use the new validation

Reply:
I see. I will delete some of the checks. Thank you for pointing this out.